cmake/CUDA_utils.cmake

```diff
@@ -40,7 +40,7 @@ endfunction()
 # List of currently used arch values
 if (${ARCH} MATCHES "aarch64-")
   # aarch64-linux
-  set(CUDA_known_archs "53" "62" "72" "75" "87")
+  set(CUDA_known_archs "72" "75" "87" "90a")
```

Comment:
Out of curiosity, why the 'a' variant in 90a?

Reply:
90a is Thor (https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/)